eng/pipelines/common/templates/runtimes/wasm-runtime-and-send-to-helix.yml

```diff
@@ -44,6 +45,7 @@ steps:
       osSubgroup: ${{ parameters.osSubgroup}}
       coreClrRepoRoot: $(Build.SourcesDirectory)/src/coreclr
       runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}
+      shouldContinueOnError: ${{ parameters.shouldContinueOnError }}
 
```

Comment:
I didn't realize we had decided to do this for the wasm runtime tests as well as android?

Reply:
This will only apply for whatever legs are part of the staging pipeline. This property will only be true if the name of the pipeline ends with staging:
```
${{ endsWith(variables['Build.DefinitionName'], 'staging') }}
```

https://github.com/dotnet/runtime/pull/45178/files#diff-39689c937bba633486600e973f3b821449f8d0f485b3eb7ee545ab8509c37992R21

So I added it to all places we send to helix to have it ready in case we need to add more platforms into this model, but this shouldn't affect current PR runs 😄

Reply:
Ah, I see. Sorry I misunderstood.